content/src/fieldConfig/business-formation.json

```diff
@@ -368,7 +368,8 @@
       "c-corporation": "Domestic Corporation",
       "s-corporation": "Domestic Corporation",
       "foreignPrefaceText": "Foreign",
-      "reviewStepLabel": "`**Business Structure**|business-structure-learn-more`",
+      "reviewStepLabel": "Business Structure",
+      "reviewStepContextualInfo": "business-structure-learn-more",
```

Comment:
can we refactor this to just be `label` and `labelContextualInfo`? It's actually used on the main page, not just the review step